src/server_manager/web_app/karma.conf.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const webpackConfig = require('../test.webpack.js');
+
+module.exports = function(config) {
+  config.set({
+    frameworks: ['jasmine'],
+    files: [
+      '**.spec.ts',
+    ],
+    preprocessors: {
+      '**.spec.ts': ['webpack'],
+    },
+    reporters: ['progress'],
+    port: 9876,
```

Comment:
Why do you need to specify a port? Can we let the system pick one? Forcing ports is usually problematic

Reply:
It's safe to remove. This is the default port value.

Karma will try to find an available port:
`If the defined port is already in use, karma will automatically increase its value in steps of 1 until a free port is found.`

Done